Gemfile.lock

```diff
@@ -5,98 +5,103 @@ PATH
       browser_sniffer (~> 1.2.2)
       jwt (~> 2.2.1)
       omniauth-shopify-oauth2 (~> 2.2.2)
-      rails (> 5.2.1, < 6.1)
+      rails (> 5.2.1, < 6.2)
```

Comment:
I wonder if we can just change this requirement to `~> 6.1` now that we know it'll be safe until v7.

Reply:
Yeah same here, I think we can probably just keep the rails requirement to be: `rails (> 5.2.1, < 7)`